app/pathogen/arbovirus/dashboard/(map)/ArbovirusMap.tsx

```diff
@@ -97,9 +98,13 @@ export function ArbovirusMap(props: ArbovirusMapProps) {
               },
             },
           ]}
-          generatePopupContent={(record) => (
-            <ArboStudyPopupContent record={record} />
-          )}
+          generatePopupContent={(input) => {
+            if(input.layerId === 'country-highlight-layer') {
```

Comment:
Hardcoded string kind of silly but will have to do for now I suppose. Might take a second look at maybe something we can do later when my TODO list is a bit thinner.